ddtrace/span.py

```diff
@@ -484,7 +484,9 @@ def pprint(self):
             ("end", "" if not self.duration else self.start + self.duration),
             ("duration", "%fs" % (self.duration or 0)),
             ("error", self.error),
+            ("sampled", self.sampled),
             ("tags", ",".join("%r:%s" % kv for kv in sorted(self.meta.items()))),
+            ("metrics", ",".join("%s:%s" % kv for kv in sorted(self.metrics.items()))),
```

Comment:
we have to do `"%r:%s"` for tags, why don't we need `%r` here for metrics?

Reply:
we have not updated the metrics typing as we did for tags